service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java

```diff
@@ -100,7 +106,103 @@ void shouldReturnEmergencySupervisionOrderLabelWhenOrderTypeEmergencySupervision
 
         RoboticsData roboticsData = roboticsDataService.prepareRoboticsData(caseData, CASE_ID);
 
-        assertThat(roboticsData).isEqualTo(expectedRoboticsData(EMERGENCY_PROTECTION_ORDER.getLabel()));
+        String expectedApplicationType = capitalize(EMERGENCY_PROTECTION_ORDER.getLabel(), EMPTY_CHAR);
```

Comment:
just provide expected text and avoid calculation like this, if EMERGENCY_PROTECTION_ORDER label will change, your test wont spot an issue but it should

btw, it looks that it is capitalised twice, here and in expectedRoboticsData